docker-compose.yml

```diff
@@ -10,6 +10,16 @@ services:
     volumes:
       - /var/run/docker.sock:/tmp/docker.sock:ro
       - "./config/yoastnginx.conf:/etc/nginx/conf.d/yoastnginx.conf"
+
+  # Mailhog:
+  mailhog:
+    container_name: "mailhog-wordpress"
+    image: mailhog/mailhog
+    logging:
+      driver: 'none'  # disable saving logs to prevent bloat
+    ports:
+      - 1025:1025 # smtp server
```

Comment:
-port 1025 this is not needed to better not expose it. it is expost on the container so can be used by the Wordpress contianers.

Reply:
Ah, yes, that seems to work. Done.